src/test/programs/federal/snap/unit.test.js

```diff
@@ -228,8 +231,69 @@ describe('SNAPhelpers', () => {
 
 
   // `SNAPhelpers.getAdjustedGross()`
+  describe('`.getAdjustedGross( timeClient )` given a time-restricted client object', () => {
+    const grossUnearnedIncome = 1;
+    let current, mock;
+
+    beforeEach(() => {
+      current = cloneDeep( defaultCurrent );
+      current.childSupportPaidOut = 100;
+      mock = jest.spyOn(cashflow, 'getGrossUnearnedIncomeMonthly');
+      mock.mockImplementation(() => grossUnearnedIncome);
+    });
+    afterEach(() => {
+      mock.mockRestore();
+    });
+
+    it('with positive raw adjusted gross income, should return that income', () => {
+      current.earned = 1000;
+      const income = current.earned + grossUnearnedIncome - current.childSupportPaidOut;
+      expect(income).toBeGreaterThan(0);
+      expect(SNAPhelpers.getAdjustedGross( current )).toEqual(income);
+    });
+
+    it('with negative raw adjusted gross income, should return zero', () => {
+      current.earned = 0;
+      const income = current.earned + grossUnearnedIncome - current.childSupportPaidOut;
```

Comment:
Is this needed?

Reply:
Its used in the following expectation to confirm the raw value is negative.